test/javascript/util/core.js

```diff
@@ -97,7 +97,7 @@ describe('Foundation core', function() {
       var name = Foundation.getFnName(B);
 
       name.should.be.a('string');
-      name.should.be.equal('');
+      name.should.be.equal('B');
```

Comment:
I seen that failed in the tests. These tests were added today in https://github.com/zurb/foundation-sites/pull/10941. It worked well with `phantomjs`.

I'm not sure of what we should expect in this case. @dragthor do you have an opinion on this ?

Reply:
phantomjs 1.9 or 2? Not sure as Chrome stable is used for the new test suite and this is more stable than the old phantomjs and should behave correctly. Not sure about the other browsers from Browserstack.

Reply:
phantomjs `v2.1.1`